src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -37,10 +37,21 @@ public partial class WindowImpl : IWindowImpl, EglGlPlatformSurface.IEglWindowGl
                 { WindowEdge.West, HitTestValues.HTLEFT }
             };
 
+        private struct SavedWindowInfo
```

Comment:
You could place it after or before other nested structs in this class to keep class layout correct.

Reply:
done.